accessing
exceptionId: anObject
	"The id of revoked exception, as reported in `exceptionThrown`.
	type: integer"
	<autoGeneratedMethod>

	exceptionId := anObject
